config/initializers/resque.rb

```diff
@@ -16,6 +18,12 @@
 }
 Resque::Mailer.excluded_environments = [] # I explicitly want this to run in tests; don't exclude them.
 
+Resque::Server.use(Rack::Auth::Basic) do |user, password|
+  username = ENV['RESQUE_WEB_HTTP_BASIC_AUTH_USER'] || 'user'
+  pw = ENV['RESQUE_WEB_HTTP_BASIC_AUTH_PASSWORD'] || 'secret'
+  [user, password] == [username, pw]
+end
```

Comment:
This seems unnecessary on dev but idk how hard it'd be to disable there safely? As long as the login is persisted it's not a huge deal.